sessions/nextjs-dynamic-routes/lotr-dynamic-routes/pages/books/index.js

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+
+export default function Books() {
+  return (
+    <>
+      <h2>Overview about all Books</h2>
+      <ul>
+        <li>
+          <Link href="/books/the-fellowship-of-the-ring">
+            The Fellowship of the Ring
+          </Link>
+        </li>
+        <li>
+          <Link href="/books/the-two-towers">The Two Towers</Link>
+        </li>
+        <li>
+          <Link href="/books/the-return-of-the-king">
+            The Return of the King
+          </Link>
+        </li>
+      </ul>
+    </>
+  );
+}
```

Comment:
rename file to `pages/volumes/index.js`

```suggestion
export default function Volumes() {
  return (
    <>
      <h2>Overview about all Volumes</h2>
      <ul>
        <li>
          <Link href="/volumes/the-fellowship-of-the-ring">
            The Fellowship of the Ring
          </Link>
        </li>
        <li>
          <Link href="/volumes/the-two-towers">The Two Towers</Link>
        </li>
        <li>
          <Link href="/volumes/the-return-of-the-king">
            The Return of the King
          </Link>
        </li>
      </ul>
    </>
  );
}
```